### PR TITLE
Adding Hifitts recipe for espnet

### PR DESCRIPTION
**Pull request by coding-phoenix-12**
Hi-Fi TTS Recipe for Espnet.

### CODE REVIEW

**Comment by sw005320** [comment]
@ftshijt, can you review this PR?

File: egs2/hifitts/tts1/conf/tuning/train_full_band_multi_spk_vits.yaml [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# This configuration is for ESPnet2 to train 44.1 kHz
```
Comment (ftshijt) [inline egs2/hifitts/tts1/conf/tuning/train_full_band_multi_spk_vits.yaml:1]: Please only PR the config related to this recipe (usually start with a single best config would be sufficient)

File: egs2/hifitts/tts1/run.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+n_shift=300
+win_length=1200
+
+opts=
+if [ "${fs}" -eq 48000 ]; then
+    # To suppress recreation, specify wav format
+    opts="--audio_format wav "
+else
+    opts="--audio_format flac "
+fi
```
Comment (ftshijt) [inline egs2/hifitts/tts1/run.sh:19]: I saw that in the readme you said the recipe is for 22.05kHz setup, but the default fs is set to 24000 and 48000 (for raw waveform)? which seems to be a conflict

**Review by ftshijt — COMMENTED** [review]
Please also remember to add your corpus to egs2/README.md

**Comment by coding-phoenix-12** [comment]
I have made the changes mentioned. Please have a look. 
Thank you

**Comment by ftshijt** [comment]
LGTM! thanks for your contribution. Could you please also fix the CI errors? After that, this PR shall be ready for merge

**Comment by coding-phoenix-12** [comment]
The CI checks on Ubuntu seem to be failing at egs2/asvspoof/spk1/run.sh . Is there something I must do to fix this?

**Comment by sw005320** [comment]
Can you fix this?
https://github.com/espnet/espnet/actions/runs/9284023148/job/25545494886?pr=5784#step:8:412

**Comment by ftshijt** [comment]
Thanks for your contribution! The current PR looks good to me~

**Pull request merged**